Gemfile.lock

```diff
@@ -74,6 +74,11 @@ GEM
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
     attribute_normalizer (1.2.0)
+    authtrail (0.1.3)
```

Comment:
the gem recommends [rack-attack](https://github.com/kickstarter/rack-attack) but that might be out of scope and definitely not required.

Reply:
Yeah, I noticed that.  Also of note, that would be good to dig into: https://ankane.org/hardening-devise and https://github.com/ankane/secure_rails